tests/testthat/test-poisson.R

```diff
@@ -0,0 +1,26 @@
+context("poisson regression")
```

Comment:
The authors of testthat now recommend not using `context()` at all. Perhaps remove it everywhere?